uk.co.compucorp.civicrm.hrcore/backstop_data/casper_scripts/contact-summary/absence/tab-entitlements.js

```diff
@@ -0,0 +1,18 @@
+'use strict';
+
+var page = require('../../../page-objects/contact-summary');
+
+module.exports = function (casper) {
+  page.init(casper).openTab('absence')
```

Comment:
Using the openTab function is not necessary because in the url itself you are mentioning the tab name, which opens the tab.